docs/services/emailservice.md

```diff
@@ -0,0 +1,66 @@
+# email service
+
+This service will send a confirmation email to the user when an order is placed.
+
+[Email service source](../../src/emailservice/)
+
+## Initialize SDK
+
+The Ruby SDK uses OpenTelemetry standard environment variables to configure
+OTLP export, resource attributes, and service name automatically. When
+initializing the OpenTelemetry SDK you can specify which auto-instrumentation
+libraries to leverage.
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
```

Comment:
The following could be included in this block to be aligned with Ruby getting started docs:
require "opentelemetry/sdk"
require "opentelemetry/exporter/otlp"
require "opentelemetry/instrumentation/sinatra"

Reply:
Yes, this is good!